src/constants/console.js

```diff
@@ -13,7 +13,15 @@ export const CONSOLE_IN_USE = 'IN_USE'
 export const CONSOLE_LOGON = 'LOGON'
 
 export const RDP_ID = 'rdp'
+// console protocols
 export const VNC = 'vnc'
-export const NO_VNC = 'noVNC'
 export const SPICE = 'spice'
 export const RDP = 'rdp'
+
+// VNC modes sent from the backend (ClientConsoleMode)
```

Comment:
ClientConsoleMode -> ClientModeVncDefault ?

Reply:
I was referring to `VncConsoleModel.ClientConsoleMode` enum but referring to config property is better.